xml/System/Convert.xml

```diff
@@ -1269,11 +1269,15 @@
         <Parameter Name="options" Type="System.Base64FormattingOptions" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="bytes">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="bytes">A read-only span of 8-bit unsigned integers.</param>
+        <param name="options">
+          <see cref="F:System.Base64FormattingOptions.InsertLineBreaks" /> to insert a line break every 76 characters, or <see cref="F:System.Base64FormattingOptions.None" /> to not insert line breaks. The default value is <see cref="F:System.Base64FormattingOptions.None" />.</param>
+        <summary>Converts the 8-bit unsigned integers inside the specified read-only span into their equivalent string representation that is encoded with base-64 digits, allowing to optionally specify whether to insert line breaks in the return value.</summary>
```

Comment:
Breaking the sentence in two because it was a bit long.

```suggestion
        <summary>Converts the 8-bit unsigned integers inside the specified read-only span into their equivalent string representation that is encoded with base-64 digits. You can optionally specify whether to insert line breaks in the return value.</summary>
```